### DISCUSSION TITLE
Need Requirement File

### DISCUSSION THREAD

**Discussion by DeadlyAbyss** [pyg-team/pytorch_geometric#9048] [Q&A]
An error occured when running \benchmark\points\point_cnn.py.
Probably because of the versions' difference in torch_cluster
Since it has many many different versions, thanks great a lot if a requirement file is provided!
![image](https://github.com/pyg-team/pytorch_geometric/assets/103022201/f17b4c7f-4049-4657-b016-b5df3a5103cf)
I'm a new starter, so i feel really sorry if it is really an easy issue...

**Comment by rusty1s** [comment]
You can generally see the latest supported versions of extension packages according to PyTorch versions here: https://data.pyg.org/whl/index.html

**Reply by rusty1s** [reply]
Will reply to any issues with `point_cnn.py` in https://github.com/pyg-team/pytorch_geometric/issues/2914. Running now.
